liquidation-monitor/src/agent.js

```diff
@@ -0,0 +1,500 @@
+          data.borrow[token.address][account.address] = BigNumber(
+            token.borrow_balance_underlying.value,
+          );
+
+          ts(
+            `User ${[account.address]
+            } borrowed ${Math.round(token.borrow_balance_underlying.value)
+            } ${data.tokens[token.address].symbol}`,
+          );
+        }
+        // Process supplies as 'cTokens'
+        if (
+          token.supply_balance_underlying !== undefined
+          && token.supply_balance_underlying.value != 0
+        ) {
+          data.supply[token.address][account.address] = BigNumber(
+            token.supply_balance_underlying.value,
+          ).dividedBy(data.tokens[token.address].exchangeRateMult);
+
+          ts(
+            `User ${[account.address]
+            } supplied ${Math.round(token.supply_balance_underlying.value)
+            } ${data.tokens[token.address].symbol
+            } ( ${Math.round(data.supply[token.address][account.address])
+            } cTokens ) `,
+          );
+        }
+        /* eslint-enable eqeqeq */
+      }); // end token loop
+      /* eslint-enable no-param-reassign */
+    }); // end account loop
+    // #endregion
+  };
+}
+
+function provideHandleTransaction(data) {
+  return async function handleTransaction(txEvent) {
+    // #region Filter logs and look for new account activity.
+    // Filter all new transactions and look for new accounts to track.
+    const borrowString = 'event Borrow(address borrower, uint256 borrowAmount, uint256 accountBorrows, uint256 totalBorrows)';
+    const exitMarketString = 'event MarketExited(address cToken, address account)';
+
+    const exitMarketEvents = txEvent.filterLog(exitMarketString);
+    exitMarketEvents.forEach((exitEvent) => {
+      ts(`${exitEvent.args.account} exited from ${exitEvent.args.account} `);
+      data.newAccounts.push(exitEvent.args.account);
+    });
+    const borrowEvents = txEvent.filterLog(borrowString);
+    borrowEvents.forEach((borrowEvent) => {
+      ts(`${borrowEvent.args.borrower} borrowed ${borrowEvent.args.borrowAmount} `);
+      data.newAccounts.push(borrowEvent.args.borrower);
+    });
+    // #endregion
+
+    // Return zero findings
+    return [];
+  };
+}
+
+function provideHandleBlock(data) {
+  return async function handleBlock(blockEvent) {
+    ts(`Starting block ${blockEvent.blockNumber}`);
+    const findings = [];
+    const { comptrollerContract, oneInchContract } = data;
+
+    // #region Add new Accounts
+    ts(
+      `Currently tracking ${Object.keys(data.accounts).length
+      } accounts and adding ${data.newAccounts.length
+      } account(s) this block. ${data.totalNewAccounts
+      } total accounts added since start.`,
+    );
+    data.newAccounts.forEach((newAccount) => {
+      const account = newAccount.toLowerCase();
+      // Initialize account. New accounts will get updated in the block section
+      /* eslint-disable no-param-reassign */
+      data.accounts[account] = {};
```

Comment:
Nit: You can eliminate the extra variable.
```suggestion
      // Initialize account. New accounts will get updated in the block section
      /* eslint-disable no-param-reassign */
      data.accounts[newAccount.toLowerCase()] = {};
```